libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -354,8 +376,36 @@ string ABIFunctions::conversionFunction(Type const& _from, Type const& _to)
 			break;
 		}
 		case Type::Category::FixedPoint:
-			solUnimplemented("Fixed point types not implemented.");
+		{
+			FixedPointType const &fromFixedType = dynamic_cast<FixedPointType const &>(_from);
+			if (toCategory == Type::Category::Integer)
+			{
+				body =
+						Whiskers("converted := <toInt>(<cleanFixed>(value))")
+						("cleanFixed", cleanupFunction(_from, false))
+						("toInt", scaleFixedFunction(fromFixedType.fractionalDigits(), 0, fromFixedType.isSigned()))
+						.render();
+			}
+			else if (toCategory == Type::Category::FixedPoint)
+			{
+				FixedPointType const &toFixedType = dynamic_cast<FixedPointType const &>(_to);
+				body =
+						Whiskers("converted := <scale>(<cleanFixed>(value))")
+						("scale", scaleFixedFunction(fromFixedType.fractionalDigits(), toFixedType.fractionalDigits(),
```

Comment:
If you break a line inside an argument list, please put all arguments on their own line (see above).